spec/models/ems_refresh/refreshers/openstack/refresh_spec_common.rb

```diff
@@ -165,11 +165,13 @@ def assert_specific_flavor
       ManageIQ::Providers::Openstack::CloudManager::Flavor.all.each do |flavor|
         expect(flavor.ext_management_system).to eq @ems
         # TODO(lsmola) expose below to Builder's data
-        expect(flavor.enabled).to               eq true
-        expect(flavor.cpu_cores).to             eq nil
-        expect(flavor.description).to           eq nil
-        expect(flavor.disk_size).to_not         be_nil
-        expect(flavor.disk_count).to            eq flavor.disk_size == 0 ? 0 : 1
+        expect(flavor.enabled).to                 eq true
+        expect(flavor.cpu_cores).to               eq nil
+        expect(flavor.description).to             eq nil
+        expect(flavor.root_disk_size).to_not      be_nil
+        expect(flavor.swap_disk_size).to_not      be_nil
+        expect(flavor.ephemeral_disk_size).to_not be_nil
```

Comment:
@Fryguy Sorry - this was my suggestion.

Similar to your suggest, I had first asked joe to populate these values. But then I realized it was just testing the vcr fixture and not our code.

So I still feel `expect(flavor.ephemeral_disk_size).to_not be_nil` or `expect(flavor.ephemeral_disk_size).to be` is the right approach. It is testing that a value was copied and not the actual value in a test text file.